docs/developer-guide/index.md

```diff
@@ -164,6 +164,17 @@ cluster you need to specify the kubeconfig to be used:
 make e2e-test KUBECONFIG=$HOME/.kube/config
 ```
 
+There are several environment variables that can be used to customize the
+e2e-tests:
+
+| Variable                   | Description                                                       | Default value |
+| -------------------------- | ----------------------------------------------------------------- | ------------- |
+| KUBECONFIG                | Kubeconfig for running e2e-tests                                  | *empty* |
+| E2E_TEST_CONFIG           | Parameterization file of e2e-tests (see [example][e2e-config-sample]) | *empty* |
+| E2E_PULL_IF_NOT_PRESENT  | Set ImagePullPolicy to IfNotPresent during e2e-tests | false |
+| E2E_TEST_FULL_IMAGE     | Run e2e-test also against the Full Image | false |
+| E2E_GINKGO_LABEL_FILTER | Ginkgo label filter to use for running e2e tests | *emptys* |
```

Comment:
nit: fix the table alignment – there is clearly effort to have it nicely aligned so make it so

Another suggestion: it's nice to have the test related variables in a separate table here -> I suggest to remove the E2E_* variables from the other table